packages/api/src/dataProviders/api/dataBretagne/DataBretagneDomaineFonctionnelAdapter.test.ts

```diff
@@ -0,0 +1,24 @@
+import DataBretagneDomaineFonctionnelAdapter from "./DataBretagneDomaineFonctionnelAdapter";
+import { DataBretagneDomaineFonctionnelDto } from "./DataBretagneDto";
+import DomaineFonctionnelEntity from "../../../entities/DomaineFonctionnelEntity";
+
+describe("DataBretagneDomaineFonctionnelAdapter", () => {
+    describe("toEntity", () => {
+        it("should convert DataBretagneDomaineFonctionnel Dto to DomaineFonctionnelEntity", () => {
+            const dto: DataBretagneDomaineFonctionnelDto = {
+                code: "code",
+                code_programme: "163",
+                label: "Label",
+            };
+            const result = DataBretagneDomaineFonctionnelAdapter.toEntity(dto);
+
+            expect(result).toBeInstanceOf(DomaineFonctionnelEntity);
+            expect(result.libelle_action).toEqual(dto.label);
+            expect(result.code_action).toEqual(dto.code);
+            // pourquoi en haut toEqual et en bas toBe ? J'ai
```

Comment:
`toBe` regarde seulement la valeur alors que `toEqual` regarde si c'est le même object "in memory".

Par sécurité on (je?) utilise uniquement `toEqual`.

Par contre en général on ne fait qu'une seule assertion par test ! Si tu veux tester toutes les valeurs de ton entitée alors peut-être un snapshot ou une comparaison d'objet en définissant un `const expected = { libelle_action: "...", code_action: "..."}`

Reply:
On peut utiliser `toBe` pour des types basiques comme `string` ou `number` en gros. Je suis moins absolue sur "un seul `expect` par test" par contre pour ce cas là oui c'est beaucoup plus lisible de faire l'`expect` sur l'objet entier plutôt que sur chaque attribut. Si tu veux laisser la possibliité qu'il y ait d'autres attributs que ceux que tu testes tu peux utiliser `toMatchObject`

Reply:
Et tu pourrais garder l'`expect` `toBeInstanceOf` par contre (pour moi)

Reply:
merci, modifs faites!